client/components/Account/DeleteAccountForm.vue

```diff
@@ -12,11 +12,12 @@ export default {
       method: 'DELETE',
       loadAccount: true,
       title: 'Delete account',
-      fields: [],
-      content: 'Deleting your account is permanent and irreversible. Proceed only if you understand these consequences.',
+      content: 'Deleting your shared account will permanently and irreversibly delete all of your shared data.<br>\
+                Proceed only if you understand these consequences.',
+      buttonClass: 'btn-danger',
```

Comment:
I think stressing that the data is shared is helpful. Maybe go a step further to say "all the shared data for {accountName}", since "your shared data" can still be ambiguous?

Reply:
sure